src/components/QueryEditor/QueryBuilder/components/QueryBuilderFilters/QueryBuilderFieldFilter.tsx

```diff
@@ -68,7 +68,7 @@ const QueryBuilderFieldFilter = ({ datasource, filter, query, indexPath, timeRan
     const setterValues = type === FilterFieldType.Name ? setFieldNames : setFieldValues
 
     setterLoading(true)
-    const list = await datasource.languageProvider?.getFieldList({ type, timeRange, field, })
+    const list = await datasource.languageProvider?.getFieldList({ type, timeRange, field, limit: 999 })
```

Comment:
Can we use const for the limit value?

Reply:
I thought we could add a configuration field for this limit in the data source settings—similar to what's done here: https://github.com/VictoriaMetrics/victoriametrics-datasource/pull/91. I'll update the PR.

Reply:
Yeah, I think it is better than hard code the value

Reply:
updated